packages/@storefront/breadcrumbs/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Added
+- SF-1248: Added a `shouldUpdate` function to the `refinement-crumbs` component.
+  - The addition of this function allows for the component to update as needed and have the latest data from the application store.
```

Comment:
I would include a note re: *why* this is necessary? For example: "Since the `field` prop does not change over the lifespan of the component, the `shouldUpdate` method is required in order to re-render in response to newly applied refinements."

Reply:
Noted, will add that in.